astroplan/tests/test_observer.py

```diff
@@ -1097,6 +1099,7 @@ def print_pyephem_moon_altaz():
     time = Time('2012-06-21 03:00:00')
     location = EarthLocation.from_geodetic(-155*u.deg, 19*u.deg, 0*u.m)
 
+    import ephem
```

Comment:
It passes because the tests only run functions that begin with `test_*` in the name.